### ISSUE TITLE
[FR]: sendmail-auth bruteforce sendmail-8.16.1

### ISSUE THREAD

**Issue by bsiege** [fail2ban/fail2ban#3030]
### Environment:

- Fail2Ban version: py38-fail2ban-0.11.2 
- OS, including release name/version :  FreeBSD STABLE-13

#### Service, project or product which log or journal should be monitored

- Name of filter or jail in Fail2Ban (if already exists) :  sendmail-auth
- Service, project or product name, including release name/version :  sendmail-8.16.1_4 
- Repository or URL (if known) : 
- Service type : 
- Ports and protocols the service is listening : 

#### failures in sense of fail2ban filter (fail2ban must match):
```
May 24 01:58:40 somehostname sm-mta[65696]: 14NNwaRl065696: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=scanner, relay=[31.210.21.53]
May 24 01:59:07 somehostname sm-mta[65815]: 14NNx65Q065815: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: scan@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=scan, relay=[31.210.20.7]
May 24 01:59:09 somehostname sm-mta[65815]: 14NNx65Q065815: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=scan, relay=[31.210.20.7]
May 24 01:59:28 somehostname sm-mta[65886]: 14NNxPRr065886: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: postmaster@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=postmaster, relay=[45.133.1.219]
May 24 01:59:31 somehostname sm-mta[65886]: 14NNxPRr065886: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=postmaster, relay=[45.133.1.219]
May 24 02:07:54 somehostname sm-mta[72797]: 14O07q4t072797: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: info@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=info, relay=bestnewacces.com [203.159.80.61] (may be forged)
May 24 02:07:55 somehostname sm-mta[72797]: 14O07q4t072797: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=info, relay=bestnewacces.com [203.159.80.61] (may be forged)
May 24 02:09:57 somehostname sm-mta[73287]: 14O09tjR073287: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: info@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=info, relay=[45.133.1.118]
May 24 02:09:58 somehostname sm-mta[73287]: 14O09tjR073287: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=info, relay=[45.133.1.118]
May 24 02:10:45 somehostname sm-mta[74467]: 14O0Ajai074467: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: info@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=info, relay=[31.210.20.173]
May 24 02:10:45 somehostname sm-mta[74467]: 14O0Ajai074467: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=info, relay=[31.210.20.173]
May 24 02:13:58 somehostname sm-mta[76688]: 14O0DuhX076688: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: admin@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=admin, relay=[45.133.1.127]
May 24 02:13:58 somehostname sm-mta[76696]: 14O0Duqd076696: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: admin@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=admin, relay=[45.133.1.218]
May 24 02:13:59 somehostname sm-mta[76688]: 14O0DuhX076688: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=admin, relay=[45.133.1.127]
May 24 02:13:59 somehostname sm-mta[76696]: 14O0Duqd076696: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=admin, relay=[45.133.1.218]
May 24 02:14:49 somehostname sm-mta[76852]: 14O0ElDe076852: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: scanner@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=scanner, relay=[45.133.1.224]
May 24 02:14:50 somehostname sm-mta[76852]: 14O0ElDe076852: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=scanner, relay=[45.133.1.224]
May 24 02:14:53 somehostname sm-mta[76856]: 14O0Ep55076856: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: postmaster@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=postmaster, relay=[31.210.20.39]
May 24 02:14:53 somehostname sm-mta[76856]: 14O0Ep55076856: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=postmaster, relay=[31.210.20.39]
May 24 02:17:04 somehostname sm-mta[79700]: 14O0H2YT079700: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: scan@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=scan, relay=[31.210.20.155]
May 24 02:17:05 somehostname sm-mta[79700]: 14O0H2YT079700: AUTH failure (DIGEST-MD5): user not found (-20) SASL(-13): user not found: unable to canonify user and get auxprops, user=scan, relay=[31.210.20.155]
May 24 02:20:30 somehostname sm-mta[81103]: 14O0KRnU081103: AUTH failure (CRAM-MD5): user not found (-20) SASL(-13): user not found: user: test@somehostname.somedomain.com property: userPassword not found in sasldb /usr/local/etc/sasldb2, user=test, relay=[45.133.1.192]

```

**Comment by sebres** [comment]
Fixed in all branches >= 0.10. Thx!

**Comment by bsiege** [comment]
Perfect!!!
